src/app/models/AdditionalResource.ts

```diff
@@ -0,0 +1,6 @@
+export interface AdditionalResource {
```

Comment:
Could consider using this interface to structure the data for the new TEP resource cards?

Reply:
It is implemented in gene-resources.component.ts.  Let me know if I misunderstood.

Reply:
`AdditionalResource` is used to structure the data for the additional resources displayed below the TREAT-AD resource section. I was wondering if the same interface could be used to structure the new TEP resource cards, so it would be easier to see the card data separate from the html tags. For example, set up the card data in the component: 

```typescript
// gene-resources.component.ts
const card1 = {
  title: 'Target Enabling Resources',
  description:
  `View the openly available TREAT-AD resources for experimental validation of ${this.gene?.gene.hgnc_symbol || this.gene?.ensembl_gene_id} in the AD Knowledge Portal.`,
  linkText: 'View Available Resources',
  link: `${this.gene?.resource_url}`,
 }
const card2 = {...}
const card3 = {...}
```
Then `card1`, `card2`, and `card3` could be used in the template, similar to how `additionalResources` is used. 

However, all good to keep things as they are! Up to you.

Reply:
Great suggestion considering the similarities.  I think for now, I will leave these as is since the TEP resources are driven off the properties of the input gene object.